### ISSUE TITLE
JetBrains: <br />s in code blocks in user messages

### ISSUE THREAD

**Issue by vdavid** [sourcegraph/cody#131]
Newlines are converted into <br /> in the user's messages (input only, not Cody's responses):

![image](https://github.com/sourcegraph/sourcegraph/assets/2552265/43399102-c590-4d7a-a216-ca87d5214a9b)

Suggested solution: make user messages plain text. In ChatGPT, user input is also just plain text, so I think users don't really expect rendered markdown in their messages. It should be an easy way to avoid formatting issues like this.

**Comment by szymonprz** [comment]
But what about the code blocks? For example if we use some recipe with the selected code the code should be displayed in the editor not as a plain text right?

**Comment by vdavid** [comment]
Great question.

Data points:
- In ChatGPT, user input is displayed as plain text
- In VS Code, we parse user input as markdown

@szymonprz, is it viable to leave the `<br />`s in normal text but not in code blocks?

**Comment by szymonprz** [comment]
Yeah it should be possible to replace the new lines not in the whole text but only in the certain type of markdown nodes
